src/appsco-app.html

```diff
@@ -4642,6 +4680,47 @@
                 this._addCustomer(customer);
                 this._notify('New customer ' + customer.name + ' was successfully added.');
             },
+
+            _onRemoveCustomersAction: function() {
+                var selectedCustomers = this.$.appscoCustomersPage.getSelectedCustomers();
+
+                if (selectedCustomers.length > 0) {
+                    this.set('_selectedCustomers', selectedCustomers);
+                    this.$.appscoCustomersRemove.toggle();
+                }
+                else {
+                    this._onHideCustomersBulkActions()
+                }
+            },
+
+            _onRemovedCustomers: function() {
+                $customersCount = this.$.appscoCustomersPage.removeCustomers(this._selectedCustomers);
+                this._onHideCustomersBulkActions();
+
+                this._notify($customersCount + ' customers were successfully removed from company.');
+            },
+
+            _onRemoveCustomersFailed: function() {
+                this.set('_selectedCustomers', []);
+                this._notify('An error occurred. Selected customers were not removed from company. Please try again.');
+            },
+
+            _onAddPartnerAdminToCustomersAction: function() {
+                var selectedCustomers = this.$.appscoCustomersPage.getSelectedCustomers();
+
+                if (selectedCustomers.length > 0) {
+                    this.set('_selectedCustomers', selectedCustomers);
+                    this.$.appscoAddPartnerAdmin.open();
+                }
+                else {
+                    this._onHideCustomersBulkActions()
+                }
+            },
+
+            _onPartnerAdminAdded: function(event) {
+                this._notify('Partner admin added successfully');
```

Comment:
'Partner administrator Branka Stanic (event.detail.admin.name) added succ. for customer Nikola Company (event.detail.customer.name)'. With doth sign at the end.